mhs/inbound/main.py

```diff
@@ -88,7 +93,9 @@ def main():
     secrets.setup_secret_config("MHS")
     log.configure_logging()
 
-    certificates = certs.Certs.create_certs_files(definitions.ROOT_DIR,
+    data_dir = pathlib.Path(definitions.ROOT_DIR) / 'data'
+
+    certificates = certs.Certs.create_certs_files(data_dir / '..',
```

Comment:
```suggestion
    certificates = certs.Certs.create_certs_files(definitions.ROOT_DIR,
```
And do other changes with this